string id
string url
string user_agent
string binary
lg_common/WindowGeometry geometry
lg_common/BrowserExtension[] extensions
lg_common/BrowserCmdArg[] command_line_args
